packages/flex-plugin-scripts/src/config/index.ts

```diff
@@ -85,7 +90,11 @@ const getConfiguration = async <T extends ConfigurationType>(
     const config = jestFactory();
 
     if (checkFilesExist(getPaths().app.jestConfigPath)) {
-      return require(getPaths().app.jestConfigPath)(config, args);
+      try {
+        return require(getPaths().app.jestConfigPath)(config, args);
```

Comment:
Why is this being wrapped? `try { ... } catch (e) { throw e }` is just the same as not doing `try/catch`